xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -123,6 +123,26 @@ static void fetchDisplayModes()
   }
 }
 
+namespace
+{
+std::string HdrTypeString(int type)
+{
+  switch (type)
```

Comment:
Maybe a std::map is better here?

Reply:
I'm not sure if it would be better, because the types are coming from Android and there's always a possibility of a new one being added.
A map wouldn't have a default fallback in that case, and would require extra logic.

Reply:
```
const std::map<int, const char*> hdrMap =
{
...
}

auto hdr = hdrMap.find(type);
if (hdr != hdrMap.end())
  return hdr.second;

return "unknown"
```

Done on my phone so it's paraphrased. The map could probably go in an anonymous namespace and this method probably wont be needed.

Just my 2c.

Reply:
Okay, that makes sense.
I'll change it. Thanks.